packages/core/src/delegation/DelegationDecoder.ts

```diff
@@ -60,7 +60,7 @@ export function decodeDelegationHierarchyDetails(
  * @param bitset The u32 number used as the bitset to encode permissions.
  * @returns The permission set.
  */
-function decodePermissions(bitset: number): PermissionType[] {
+function permissionsFromChain(bitset: number): PermissionType[] {
```

Comment:
if it's not a Codec, it's not really a `fromChain` is it?

Reply:
i'd say if we keep the name we change the input to `typeof delegationNode.details.permissions`

Reply:
Done in https://github.com/KILTprotocol/sdk-js/pull/613/commits/d5f5a4ba4718263b53a0c6d794e7e93d9b2a95bc